largest-number-from-array.playground/Contents.swift

```diff
@@ -5,9 +5,7 @@ import Cocoa
 import XCTest
 
 func largestNumberComposed(from: Array<Int>) -> Int {
-    return Int(from.map{String($0)}
-        .sort({!$0.hasPrefix($1) && ($1.hasPrefix($0) || $0>$1)})
-        .joinWithSeparator(""))!
+   return Int(from.map{String($0)}.sort{$0+$1>$1+$0}.joinWithSeparator(""))!
```

Comment:
Here is a cute solution (also mentioned by @Sarp Centel). The idea is to compare the concatenation of two numbers with the reverse concatenation of those numbers.
For example, to compare 56 and 5, we'd do a regular lexicographic comparison of 565 to 556. Since 565 > 556, we'll say that 56 is "bigger" than 5, and should come first. Similarly, comparing 54 and 5 means we'll test 545 < 554, which tells us that 5 is "bigger" than 54.

I am investigating why it is really slow 👎